invenio_app_ils/ui/src/invenio_app_ils/common/urls.js

```diff
@@ -5,6 +5,7 @@ const FrontSiteBase = '/';
 
 export const FrontSiteURLS = {
   home: FrontSiteBase,
+  booksSearch: `${FrontSiteBase}books`,
```

Comment:
Can I propose something? 
```
* recordsSearch: `${FrontSiteBase}search`
* recordsDetails: `${FrontSiteBase}records/:documentPid`
```
And so on for the entire frontsite, not for the backoffice. This is because we also have standards, thesis and other types of `Document`.

WDYT?

Reply:
I now renamed every instance of book, replacing it with record :)

Reply:
wow, a lot of work with renaming! In general good job!
I will put my naming police hat again though. The link as records was meant to be visible for the end user (as for them we have not only books but also standards  and proceedings, but they are not aware of items). For us as developers in the code would be better if we name it as documents, because items are also records, and someone might be confused by that. But please before you rename again let's see the other's opinions!

Reply:
Agree

Reply:
I agree too. Will be easier to understand the code if we use the developer terminology in the code but displays a more user-friendly name to the user.